### DISCUSSION TITLE
Stand aus Github ins SVN

### DISCUSSION THREAD

**Discussion by HomeAutoUser** [RFD-FHEM/RFFHEM#1045] [Ideas]
Die Benutzer könnten nach fast 2 Jahren ja mal wieder einen neuen Stand erhalten.

**Comment by elektron-bbs** [comment]
Das ist ja jetzt geschafft :-)
Ich vermute, das in der 00_SIGNALduino.pm auf Github übersehen wurde, die SDUINO_VERSION anzupassen:

```
# $Id: 00_SIGNALduino.pm v3.5.4 2022-01-23 10:53:46Z sidey79 $
  SDUINO_VERSION                  => '3.5.2+20220123',  # Datum wird automatisch bei jedem pull request aktualisiert
```

Da im SVN jetzt die FSK-Protokolle enthalten sind, wäre es sicher als nächstes angebracht, die SIGNALduino-Version dev-r3.5_xFSK in den Master-Branch zu nehmen und ein neues Release zu erstellen.

**Comment by sidey79** [comment]
SDUINO_VERSION würde angepasst.
Wo hast Du die Abweichung gesehen?

**Comment by elektron-bbs** [comment]
Ich habe vor ca. 3 Stunden ein update getätigt und sehe da in der 00_SIGNALduino.pm diese Differenz:

```
# $Id: 00_SIGNALduino.pm v3.5.4 2022-01-23 10:53:46Z sidey79 $
                              ^
  SDUINO_VERSION                  => '3.5.2+20220123',  # Datum wird automatisch bei jedem pull request aktualisiert
                                          ^
```

**Comment by sidey79** [comment]
Wir reden über SVN?
Da ist die 3.5.3 hinterlegt:

https://svn.fhem.de/trac/browser/trunk/fhem/FHEM/00_SIGNALduino.pm

**Comment by elektron-bbs** [comment]
Nein, über Github Branch master.
Da im SVN eine 3.5.3 liegt, nahm ich an, wir machen hier mit einer 3.5.4 weiter, so wie es auch in der Kopfzeile steht:

$Id: 00_SIGNALduino.pm v3.5.**4**

In SDUINO_VERSION steht aber noch:

SDUINO_VERSION                  => '3.5.**2**

**Reply by sidey79** [reply]
Das ist ein Fehler, würde durch einen PR wieder auf 3.5.2 geändert :(

3.5.4 wäre korrekt

**Comment by elektron-bbs** [comment]
3.5.4 ist jetzt im SVN.
3.5.5 aktuell auf GitHub.

**Comment by sidey79** [comment]
@elektron-bbs 

Ich hab mal 3.5.6 vorbereitet.... es fehlt noch das Changelog und man müsste noch mal durchschauen wo noch 3.5.5 steht :)

**Comment by elektron-bbs** [comment]
Mhmm, bei mir steht überall schon:
`versionmodul | 3.5.7+20250219
`
bzw.
`versionmodul | 3.5.7+20260116
`

**Reply by sidey79** [reply]
Gut erkannt. Ich habe eine 3.5.7 gemacht und ins SVN gepusht.

@HomeAutoUser   
Für `14_SD_UT` bist Du als alleiniger maintainer hinterlegt. Die Datei müsste im SVN noch aktualisiert werden. Alle anderen geänderten Dateien durfte ich machen.

**Reply by HomeAutoUser** [reply]
@sidey79 
Ich habe es zur Kenntnis genommen und werde die aktuelle Version hochladen.

**Reply by HomeAutoUser** [reply]
@sidey79 , @elektron-bbs 
soeben erledigt https://svn.fhem.de/trac/log/trunk/fhem/FHEM?rev=30754
